### PR TITLE
Issue #SB-18656 feat: Showing certificates of Quiz participant in profile page implementation

### PR DESCRIPTION
**Pull request by sudip-mukherjee**

### CODE REVIEW

File: src/app/client/src/app/plugins/profile/services/profile/profile.service.ts [hunk 1]
```diff
@@ -125,4 +125,12 @@ export class ProfileService {
     };
     return this.learnerService.post(options);
   }
+
+  public fetchCertificates(request) {
```
Comment (vinukumar-vs) [inline src/app/client/src/app/plugins/profile/services/profile/profile.service.ts:129]: This method should go to CertRegService & call directly from component

File: src/app/routes/certRegRoutes.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+const permissionsHelper = require('../helpers/permissionsHelper.js')
+const envHelper = require('../helpers/environmentVariablesHelper.js')
+const certRegURL = envHelper.LEARNER_URL
+const reqDataLimitOfContentUpload = '50mb'
+const proxy = require('express-http-proxy')
+const logger = require('sb_logger_util_v2')
+
+module.exports = function (app) {
+
+    app.all('/certreg/v1/certs/search',
```
Comment (vinukumar-vs) [inline src/app/routes/certRegRoutes.js:11]: Define proxy object on top & use that object.

File: src/app/client/src/app/plugins/profile/components/profile-page/profile-page.component.ts [hunk 3]
```diff
@@ -131,6 +134,29 @@ export class ProfilePageComponent implements OnInit, OnDestroy, AfterViewInit {
     });
   }
 
+  getOtherCertificates(userId) {
+    const request = {
+      request: {
+        query: {
+          match_phrase: {
+            'recipient.id': userId
+          }
```
[the remark this answers is not in the document]
Comment (sudip-mukherjee) [reply src/app/client/src/app/plugins/profile/components/profile-page/profile-page.component.ts]: done with the existing one

File: src/app/client/src/app/plugins/profile/services/profile/profile.service.ts (hunk 1 above)
Comment (sudip-mukherjee) [reply src/app/client/src/app/plugins/profile/services/profile/profile.service.ts]: moved

File: src/app/routes/certRegRoutes.js (hunk 2 above)
Comment (sudip-mukherjee) [reply src/app/routes/certRegRoutes.js]: done

**Pull request merged**